tests/test_sanity.py

```diff
@@ -116,8 +113,9 @@ def test_dotnet_trace(
     application_pid: int,
     assert_collapsed: AssertInCollapsed,
     gprofiler_docker_image: Image,
+    profiler_state: ProfilerState,
 ) -> None:
-    with DotnetProfiler(1000, 3, Event(), str(tmp_path), False, False, CPU_PROFILING_MODE, "dotnet-trace") as profiler:
```

Comment:
Unrelated - but I noticed here that `DotnetProfiler` has the arguments order not in sync with e.g `RbSpyProfiler(1000, 3, Event(), str(tmp_path), False, CPU_PROFILING_MODE, False, "rbspy")`.

Please make sure, after your change now, that the order is synced across all extenders of `ProfilerInterface`.

Reply:
Now every profiler has order: frequency, duration, profiler_state, profiler_specific_parameters